nservicebus/messaging/data-distribution.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Data distribution
+summary: Why asynchronous messaging is not a good fit for data distribution scenarios
+reviewed: 2020-05-21
+redirects:
+- nservicebus/azure/data-distribution
+- samples/routing/data-distribution
+---
+
+Data distribution refers to a pattern where asynchronous messages (e.g. using NServiceBus) are used to deliver data to multiple targets in some coordinated fashion.
+
+A common example of data distribution scenario is having cached data on multiple scaled-out web servers, and then attempting to use messages delivered to all web servers in a cache-busting fashion, so that each web server drops its current cache entries, and then is forced to retrieve fresh data from the database.
```

Comment:
```suggestion
A common example of a data distribution scenario is having cached data on multiple scaled-out web servers, and then attempting to use messages delivered to all web servers in a cache-busting fashion, so that each web server drops its current cache entries, and then is forced to retrieve fresh data from the database.
```